packages/cli/src/commands/env/rm.ts

```diff
@@ -0,0 +1,67 @@
+import { prompt } from 'inquirer'
+import * as api from '../../rest/api'
+import { Flags, Args } from '@oclif/core'
+import { AuthCommand } from '../authCommand'
+
+export default class EnvRm extends AuthCommand {
+  static hidden = false
+  static description = 'Remove environment variable via "checkly env rm <key>".'
+
+  static flags = {
+    force: Flags.boolean({
+      char: 'f',
+      description: 'Force to skip the confirmation prompt.',
+      default: false,
+    }),
+  }
+
+  static args = {
+    fileArgs: Args.string({
+      name: 'key',
+      required: true,
+      description: 'Environment variable key to remove.',
+    }),
+  }
+
+  static strict = false
+
+  async run (): Promise<void> {
+    const { flags, argv } = await this.parse(EnvRm)
+    const { force } = flags
+    const args = argv as string[]
```

Comment:
This is the older style, you can now parse args like flags

const {args} = await this.parse(EnvRm)
this.log(`running my command with args: ${args.fileArgs}`)

Reply:
Sounds good 👍 

I just re-worked how arguments are handled, since Oclif has nice support for multi arguments. 

Before, we were simply taking the args as a variable lenght array.  The help message was:
```
$ ./bin/run env add --help
Add environment variable via "checkly env add <key> <value>".

USAGE
  $ checkly env add [FILEARGS] [-l]

ARGUMENTS
  FILEARGS  Arguments to add environment variable <key> <value>.

FLAGS
  -l, --locked  Indicate that the environment variable will be locked.

DESCRIPTION
  Add environment variable via "checkly env add <key> <value>".
  ```
  
Now we specify the actual expected arguments to Oclif. The help message is a bit more detailed IMO:
  
 ```
  $ ./bin/run env add --help
Add environment variable via "checkly env add <key> <value>".

USAGE
  $ checkly env add [KEY] [VALUE] [-l]

ARGUMENTS
  KEY    Environment variable key.
  VALUE  Environment variable value.

FLAGS
  -l, --locked  Indicate that the environment variable will be locked.

DESCRIPTION
  Add environment variable via "checkly env add <key> <value>".
```

Oclif will also validate that the correct number of args are passed, so we can remove some validation code.